src/commands.rs

```diff
@@ -0,0 +1,95 @@
+use clap::Subcommand;
+use strum::{Display as StrumDisplay, EnumString};
+
+/// An enum of commands that need to be passed to pnpm
+#[derive(Debug, PartialEq, EnumString, StrumDisplay, Subcommand, Clone)]
```

Comment:
Instead of `Display` (which copies and allocates an owned string), you should have used `strum::AsRefStr`.